libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -714,6 +783,16 @@ export class RushConfiguration {
       )
     );
 
+    this.tempSubspaceShrinkwrapFileName = (subspaceName: string): string => {
```

Comment:
Why is this function defined here instead of being declared as a normal member function?

Reply:
Changed to a normal member function